packages/documentation-site/patternfly-docs/pages/home.js

```diff
@@ -1,278 +1,317 @@
-import React from 'react';
-import { Link } from '@patternfly/documentation-framework/components';
+import React from "react";
+import {Link} from "@patternfly/documentation-framework/components";
 import {
+  Button,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardFooter,
+  Gallery,
+  GalleryItem,
+  Divider,
   Flex,
   FlexItem,
   Grid,
-  GridItem,
   PageSection,
-  PageSectionVariants,
   Title,
-  TextContent,
   Text,
-  TextVariants
-} from '@patternfly/react-core';
-import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
-import GithubIcon from '@patternfly/react-icons/dist/esm/icons/github-icon';
-import orb from '../images/pf4-logo.svg';
-import accessible from '../images/accessible.svg';
-import modular from '../images/modular.svg';
-import scale from '../images/scale.svg';
-import consistent from '../images/consistent.svg';
-import open from '../images/open.svg';
-import { AutoLinkHeader } from '@patternfly/documentation-framework/components/autoLinkHeader/autoLinkHeader';
-import './home.css';
+  TextVariants,
+} from "@patternfly/react-core";
+import ArrowRightIcon from "@patternfly/react-icons/dist/esm/icons/arrow-right-icon";
+import ExternalLinkAltIcon from "@patternfly/react-icons/dist/esm/icons/external-link-alt-icon";
+import lightshowcase from "../images/light-showcase.svg";
+import darkshowcase from "../images/dark-showcase.svg";
```

Comment:
These files are pretty huge - just over 1mb each. They should export really well as PNGs, and once sized down (they're [too big](https://patternfly-org-pr-3573-site.surge.sh/images/34a87f6c.svg) now), I usually run images through a tool called [ImageOptim](https://imageoptim.com/mac) - it's free and does a good job. Or the designer that provided them may be able to optimize them for the web, too.

Reply:
@lboehling shared these images as pngs as well... I ran them though ImageOptim which reduced them to 373 KB